pipeline/pipeline.go

```diff
@@ -81,18 +81,69 @@ func (r *Runtime) Run() error {
 	return r.err
 }
 
-//
-//
-//
+// Updates the current status of a step
```

Comment:
this is also unrelated :sweat: 

can we pick out this changes first? :)

Reply:
I'll seperate it once I have a minute